.github/workflows/ci.yml

```diff
@@ -94,7 +94,7 @@ jobs:
       - run: yarn lint
         # Make sure no tests are skipped with "focused" tests.
       - run: |
-          ! find tensorboard/webapp -type f -regex '.*_test.ts$' -exec grep -E 'f(it|describe)\(' {} +
+          ! git grep -E 'f(it|describe)\(' 'tensorboard/webapp/*_test.ts' 'tensorboard/plugins/*_test.ts'
```

Comment:
Fine with me if this is what you want, though if we’re cherry-picking
two directories it’s a bit unclear to me why we don’t just run against
all of `*_test.ts`.